<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>get_task_range</class>
 <widget class="QMainWindow" name="get_task_range">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDateTimeEdit" name="min_timeEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>194</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="max_timeEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>194</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="OK_button">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>240</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLabel" name="error_range_warning">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>190</y>
      <width>45</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
